test/bin/run-integration-tests.sh

```diff
@@ -48,6 +48,7 @@ PYTHONPATH="$( pwd )" \
     python test/bin/inject_api_key.py admin@user.com \
     "XZpXI40Uk85eozjQkU1zHJ6yZHpix+j0mo1TMeGZ4dPzIqVPVGPmyfeK"
 
+PYTHONPATH="$( pwd )" \
```

Comment:
FYI -- We pulled out the access log URI, but this line is still required for validating the access type. I think that's fine.